src/main/java/fi/metatavu/kuntaapi/server/index/IndexableNewsArticle.java

```diff
@@ -0,0 +1,129 @@
+package fi.metatavu.kuntaapi.server.index;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+public class IndexableNewsArticle implements Indexable {
+
+  @Field (index = "not_analyzed", store = true, type = "long")
+  private Long orderIndex;
+  
+  @Field (index = "not_analyzed", store = true, type = "integer")
+  private Integer orderNumber;
+  
+  @Field (index = "not_analyzed", store = true)
+  private String slug;
+  
+  @Field (index = "not_analyzed", store = true, type = "date")
+  private OffsetDateTime published;  
+
+  @Field (index = "not_analyzed", store = true)
+  private String newsArticleId;
+  
+  @Field (index = "not_analyzed", store = true)
+  private String organizationId;
+  
+  @Field
+  private String title;
+
+  @Field
+  private String newsAbstract;
+
+  @Field
+  private String contents;
+  
+  @Field (index = "not_analyzed")
+  private List<String> tags;
+  
+  @Override
+  public String getId() {
+    return newsArticleId;
+  }
+
+  @Override
+  public String getType() {
+    return "newsarticle";
+  }
+
+  @Override
+  public Long getOrderIndex() {
+    return orderIndex;
+  }
+  
+  public void setOrderIndex(Long orderIndex) {
+    this.orderIndex = orderIndex;
+  }
+  
+  public Integer getOrderNumber() {
+    return orderNumber;
+  }
+  
+  public void setOrderNumber(Integer orderNumber) {
+    this.orderNumber = orderNumber;
+  }
+  
+  public void setSlug(String slug) {
+    this.slug = slug;
+  }
+  
+  public String getSlug() {
+    return slug;
+  }
+
+  public OffsetDateTime getPublished() {
+    return published;
+  }
+
+  public void setPublished(OffsetDateTime published) {
+    this.published = published;
+  }
+
+  public String getNewsArticleId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 39. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)